entrypoint.sh

```diff
@@ -59,12 +60,14 @@ if [ $(mysql -N -s -u $MYSQL_USER --password=$MYSQL_PASSWORD -e \
         table_schema='$MYSQL_DATABASE';") -ge 1 ]; then
     echo "Database already created... skipping creation..."
 else
+echo "creating DB"
   import_empty_db "$MYSQL_USER" "$MYSQL_PASSWORD" "$MYSQL_DATABASE" "$CTF_PATH" "prod"
 fi
 
 # Configuring settings.ini
 cat "$CTF_PATH/settings.tmpl.ini" \
   | sed "s/MYSQL_PORT/$MYSQL_PORT/g" \
+  | sed "/^DB_HOST/ s/mysql/$MYSQL_HOST/" \
```

Comment:
I'd rather update [settings.tmpl.ini](https://github.com/AlexGaspar/docker-fbctf/blob/master/templates/settings.tmpl.ini#L3) and use the same syntax everywhere.

Also if we do that, I'd like to have a fallback on a default value for the linked case :)